src/Microsoft.AspNetCore.BuildTools.ApiCheck/ApiListingComparer.cs

```diff
@@ -76,6 +76,24 @@ private void CompareMembers(TypeDescriptor type, TypeDescriptor newType, List<Ap
                     }
                 }
             }
+
+            if (type.Kind == TypeKind.Interface && type.Members.Count < newType.Members.Count)
```

Comment:
I believe that would have been covered by the various other checks already.

This check needs to exist because I think at this point we know that everything that was in InterfaceV1 is in InterfaceV2, but we don't know about new stuff in InterfaceV2.